R/Rules-class.R

```diff
@@ -95,10 +95,11 @@ NextBestMTD <- function(target, derive) {
 #' Otherwise, the highest admissible dose is the next recommended dose.
 #'
 #' @slot target (`numeric`)\cr the target toxicity interval (limits included).
+#'   It is used to filter probability samples.
```

Comment:
this we don't need. it is used to calculate target probabilities. the filtering you mention and we discussed yesterday is a technical implementation detail not relevant here